en/middleware/micrortps.md

```diff
@@ -230,13 +238,13 @@ In order to install ROS Melodic and ROS2 Bouncy on a Ubuntu 18.04 machine, follo
 1. [Install ROS Melodic](http://wiki.ros.org/melodic/Installation/Ubuntu)
 2. [Install ROS2 Bouncy](https://index.ros.org/doc/ros2/Linux-Install-Debians/)
 
-For the ROS2 case, so the package properly generates the IDL files, the following should be installed:
+Install the following component to ensure that the package properly generates the IDL files:
```

Comment:
FYI, I removed for "ROS2" case because this should also be done for ROS1 right?

Reply:
Well not exactly. The ROS2 side is the only one that needs the IDL's generated (note it generates IDL's not only for fast-RTPS but also for other RTPS protocols as openSplice). And the reason is that the micrortps_agent requires those. The only thing that you require generating for ROS1 is the msg headers and sources. Though of course you still need ROS2 (and ros1_bridge) to be able to bridge with the PX4 client.

Reply:
Thanks for clarification. The text is still IMO better, so not planning to change.